payment_payfip/controllers/main.py

```diff
@@ -2,63 +2,63 @@
 import pprint
 import werkzeug
 
-from odoo import _, http
+from odoo import http, exceptions
 from odoo.http import request
 
-from odoo.addons.payment.models.payment_acquirer import ValidationError
+# from odoo.addons.payment.models.payment_acquirer import ValidationError
 
 _logger = logging.getLogger(__name__)
 
 
 class PayFIPController(http.Controller):
-    @http.route('/payment/payfip/pay', type='http', auth='none', methods=['GET', 'POST'], csrf=False)
+    @http.route("/payment/payfip/pay", type="http", auth="none", methods=["GET", "POST"], csrf=False)
     def payfip_pay(self, **post):
-        reference = post.pop('reference', False)
-        amount = float(post.pop('amount', 0))
-        return_url = post.pop('return_url', '/shop/payment/validate')  # /payment/process after v11
-        tx = request.env['payment.transaction'].sudo().search([('reference', '=', reference), ('amount', '=', amount)])
-        if tx and tx.acquirer_id.provider == 'payfip':
+        reference = post.pop("reference", False)
+        amount = float(post.pop("amount", 0))
+        return_url = post.pop("return_url", "/")
```

Comment:
```suggestion
        return_url = post.pop("return_url", "/payment/process")
```
url créée par Odoo redirigeant automatiquement sur la facture avec un message type "Paiement validé"